### PR TITLE
Update release scripting and install docs

### PR DESCRIPTION
**Pull request by VJftw**

### CODE REVIEW

File: scripts/publish-pre-release.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ plz build //cmd/dracon:dracon
 
 github_api_version="application/vnd.github.v3+json"
 github_owner="thought-machine"
-github_repo="dracon-private"
```
Comment (steakunderscore) [inline scripts/publish-pre-release.sh]: Eeek.. I forgot to sed this! Maybe we should do that in a separate PR for the whole repo.

Comment (VJftw) [reply scripts/publish-pre-release.sh]: I did a `git grep` for `private` and nothing else came up apart from the documentation for trying out on a private github repo :)
